### PR TITLE
EFRS-772: Confirmation message about deleting the user contains an invalid data

### PR DESCRIPTION
**Pull request by Sisha3342**

### CODE REVIEW

File: ui/src/app/features/user-list/user-list.component.ts [hunk 1]
```diff
@@ -89,14 +89,15 @@ export class UserListComponent implements OnInit, OnDestroy {
...
-              entityType: this.translate.instant('users.system-user'),
+              entityType: this.translate.instant('users.user'),
               entity: deletion.userToDelete,
               options: [
                 { name: email, value: 'deleter' },
                 { name: this.orgOwnerEmail, value: 'owner' },
               ],
               isOrganizationOwner: email === this.orgOwnerEmail,
               seletedOption: this.seletedOption,
+              removeFromSystem: true,
```
Comment (isobolev) [inline ui/src/app/features/user-list/user-list.component.ts:100]: please rename to `isSystemUser`

**Review by isobolev — COMMENTED** [review]
looks good, just minor comment

File: ui/src/app/features/delete-dialog/delete-dialog.component.html [hunk 2]
```diff
@@ -14,46 +14,45 @@
...
   -->
 
-<h1 mat-dialog-title>{{'org_users.delete_dialog.title' | translate}}
-  {{data.entityType === 'system-user' ? 'user' : data.entityType }}</h1>
+<h1 mat-dialog-title>{{'org_users.delete_dialog.title' | translate: {type: data.entityType} }}</h1>
 <div mat-dialog-content>
   <ng-container [ngSwitch]="data.entityType.toLowerCase()">
     <ng-container *ngSwitchCase="'user'">
-      <p>{{'app_users.delete_dialog.confirmation_question' | translate :
+      <p *ngIf="!data.removeFromSystem; else notFromSystem">{{'app_users.delete_dialog.confirmation_question' | translate :
```
Comment (isobolev) [inline ui/src/app/features/delete-dialog/delete-dialog.component.html:21]: removeFromSystem is not longer defined. isSystemUser should be used

**Review by isobolev — APPROVED** [review]

**Comment by Sisha3342** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Sisha3342** [comment]
recheckcla

**Comment by Sisha3342** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**